packages/form-builder/src/blocks.json

```diff
@@ -25,6 +25,19 @@
                     "setPrice": "100",
                     "customAmount": "true",
                     "customAmountMin": 1,
+                    "recurringBillingPeriodOptions": [
+                        "day",
+                        "week",
+                        "month",
+                        "year"
+                    ],
+                    "recurringBillingPeriod": "month",
+                    "recurringBillingInterval": 1,
+                    "recurringPeriod": "preset",
+                    "recurringDonationChoice": "admin",
+                    "recurringEnabled": false,
+                    "recurringLengthOfTime": "0",
+                    "recurringOptInDefault": true,
```

Comment:
@kjohnson fyi I had to add these or else the block would break when toggling the recurring option.  I added a helper to reference the blocks.json values as the default attributes see here:

https://github.com/impress-org/givewp-next-gen/pull/128/files#diff-82ae36f5a7ccf1e5e32c1ec6803e32e8e0dc9071c9fc8a3aef39c5f64bae9257R23

I can see this being problematic in the future when adding new features. 
 We might want to re-think how we're loading the blocks in the editor & their default values.